frontend/components/vault/Vault.vue

```diff
@@ -188,7 +188,7 @@ export default Vue.extend({
             default: null,
         },
         vaultId: {
-            type: String,
+            type: Number,
```

Comment:
Since it will come from a url, this should still be a string (which you will parse inside the container in order to fetch actual vault). If the user open `?vault=test` it should still see `test` on the vault page, not `Vault #NaN`